resolvers/event.js

```diff
@@ -34,15 +34,16 @@ const eventResolvers = {
       }),
   },
   Event: {
-    createdBy: (obj, args, context, info) => db.user.findByPk(obj.createdBy),
+    createdBy: (obj) => db.user.findByPk(obj.createdBy),
   },
   Mutation: {
     addEvent: async (parent, args) => {
       // Check if createdBy is valid
-      const user = await db.user.findByPk(args.createdBy);
-      if (!user) {
-        throw new Error("Invalid user ID");
-      }
+      await db.user.findByPk(args.createdBy).then((user) => {
+        if (!user) {
+          throw new Error('Invalid user ID: ' + args.createdBy);
```

Comment:
added the invalid value to error messages for this file - [we have a ticket to do this for all other error messages](https://www.notion.so/uwblueprintexecs/resolver-cleanup-ad3e0ae00db24983ac0c0722dd6c6400)